src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,366 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Handles UI aspects of the duke.Duke chatbot.
+ *
+ * @author Yuvaraj Kumaresan
+ */
+public class Ui {
+
+    /**
+     * The storage used to interact with the user.
+     */
+    protected Storage storage;
+
+    /**
+     * Constructor.
+     *
+     * @param storage The storage used to get item to interact with user.
+     */
+    public Ui(Storage storage) {
+        this.storage = storage;
+    }
+
+    String divider = "--------------------------------------------------" +
+            "--------------------------------------------------------------";
+
+    /**
+     * Handles UI output for list keyword.
+     */
+    public void list() {
```

Comment:
Perhaps you can change the naming to a verb instead ? For eg: 
```suggestion
    public void showList() {
```